input/postgres/explain.go

```diff
@@ -44,40 +44,54 @@ func RunExplain(server state.Server, inputs []state.PostgresQuerySample, collect
 			logger.PrintVerbose("Could not connect to %s to run explain: %s; skipping", dbName, err)
 			continue
 		}
+		useHelper := statsHelperExists(db, "explain")
+		if useHelper {
+			logger.PrintVerbose("Found pganalyze.explain() stats helper")
+		} else {
+			logger.PrintInfo("Warning: pganalyze.explain() helper function not found. Please set up" +
```

Comment:
+1 maybe we should not even attempt to run the EXPLAIN if the helper doesn't exist and we're not superuser? (e.g. always return an error in the query explain error text, but not causing an unnecessary database log entry saying we couldn't run the EXPLAIN)

Reply:
I kinda feel like granting the pganalyze user SELECT permissions directly is a reasonable alternative approach: it's pretty clear that you're giving us read access in that case, so if that's what you want to do, I don't see a reason to get in your way. Thoughts?

Reply:
Ah, yeah, thats a fair point - should we only display the warning in the case of a permission error then?

Reply:
Also I just realized, since this is a warning we can use the `PrintWarning` helper, instead of `PrintInfo` (doesn't make much of a difference in practice)

Reply:
>...should we only display the warning in the case of a permission error then?

Yes, good call.

>Also I just realized, since this is a warning we can use the PrintWarning helper, instead of PrintInfo

I think I got `PrintInfo` from [here](https://github.com/pganalyze/collector/blob/7efb8e0095cde47913511528439c75384d3dba1b/input/postgres/buffercache.go#L80), but makes sense, I'll update.